web/templates/invite_user_modal.hbs

```diff
@@ -29,7 +29,9 @@
         </div>
         <div id="invitee_emails_container">
             <label for="invitee_emails">{{t "Emails (one on each line or comma-separated)" }}</label>
-            <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
+            <div  class="pill-container">
```

Comment:
This whitespace was non-standard, I took care of it, but please be careful about this sort of detail, as it makes it considerably harder to read code if we let things like that slip into the code base.

``` diff
diff --git a/web/templates/invite_user_modal.hbs b/web/templates/invite_user_modal.hbs
index a12cda198e..7c548aa1a9 100644
--- a/web/templates/invite_user_modal.hbs
+++ b/web/templates/invite_user_modal.hbs
@@ -29,7 +29,7 @@
         </div>
         <div id="invitee_emails_container">
             <label for="invitee_emails">{{t "Emails (one on each line or comma-separated)" }}</label>
-            <div  class="pill-container">
+            <div class="pill-container">
                 <div class="input" contenteditable="true"></div>
             </div>
         </div>
```

Reply:
sorry, will be careful from now on